src/frame_buffer.rs

```diff
@@ -67,6 +74,18 @@ impl<'a> FrameBuffer<'a> {
         self.dimensions
     }
 
+    /// Return the origin of the frame buffer.
+    pub fn origin(&self) -> (i32, i32) {
+    	self.origin
+    }
+
+    /// Return the offset of the first display window pixel with reference to the first data window pixel
+    pub fn origin_offset<T: Sized>(&self) -> isize {
```

Comment:
Is there a good reason for this to be public?

Reply:
It can be used as an index to the byte of the "first pixel" at 0,0 in the frame_buffer pixel data by the client app.

However, you are right as the way it is now it has no much use. In fact, now that I think of it, this should actually be split into two functions. A public one which returns the offset to the pixel (in pixels) and one which should

Reply:
> A public one which returns the offset to the pixel (in pixels)

What would an external caller do with this?

Reply:
Just added an example in `negative_window.rs`

Reply:
```
let origin_offset = {
        let mut fb = FrameBufferMut::new_with_origin(x, y, width, height);
        println!("Loading buffer as {}x{}", width, height);
        fb.insert_channels(&[("R", 0.0), ("G", 0.0), ("B", 0.0)], &mut pixel_data);
        exr_file.read_pixels(&mut fb).unwrap();
        exr_file.read_pixels(&mut fb).unwrap();
        fb.origin_offset() as usize
    };
    // check the pixel value at (0,0). "==" is
    assert!(f32::abs(pixel_data[origin_offset].0.to_f32() - 0.5f32) < 0.0001f32);
```